spec/assigner_spec.rb

```diff
@@ -4,15 +4,17 @@
   context 'named special cases' do
     context 'weekly meetings' do
       it 'should assign the meetings' do
-        a = Assigner.new '92345678: internal weekly meeting'
+        a = Assigner.new '92345678: internal weekly meeting : : 2017-04-25T09:02:30+00: 00'
```

Comment:
![image](https://cloud.githubusercontent.com/assets/326935/25424421/96a75fb4-2a68-11e7-9ad3-a4669a9d9e7b.png)

=> just use the last part as date